pkg/agent/datamodel/types.go

```diff
@@ -133,6 +133,7 @@ const (
 	AKSUbuntuContainerd1804Gen2    Distro = "aks-ubuntu-containerd-18.04-gen2"
 	AKSUbuntuGPUContainerd1804     Distro = "aks-ubuntu-gpu-containerd-18.04"
 	AKSUbuntuGPUContainerd1804Gen2 Distro = "aks-ubuntu-gpu-containerd-18.04-gen2"
+	AKSMariner                     Distro = "aks-mariner"
```

Comment:
we should add at least one more dimension for future mariner skus that will have different distro name and vhd

Reply:
Changed the tag to AKSMarinerV1 and added a AKSMarinerGPUV1 tag.